### ISSUE TITLE
long_term_forecast, feature='MS', 使用inverse_transform方法时报错。

### ISSUE THREAD

**Issue by Gczmy** [thuml/Time-Series-Library#254]
您好，我在测试我的数据集时，使用的是long_term_forecast, feature='MS'，期望多输入单输出，在exp_long_term_forecast.py文件处报错张量的形状不匹配，源代码：
https://github.com/thuml/Time-Series-Library/blob/a9db7887779500f9fd8a7278eacbf46dec4a6654/exp/exp_long_term_forecasting.py#L216-L224

我尝试修改该部分代码：
```python
f_dim = -1 if self.args.features == 'MS' else 0
outputs = outputs[:, -self.args.pred_len:, :]
batch_y = batch_y[:, -self.args.pred_len:, :].to(self.device)
outputs = outputs.detach().cpu().numpy()
batch_y = batch_y.detach().cpu().numpy()
if test_data.scale and self.args.inverse:
    shape = outputs.shape
    outputs = test_data.inverse_transform(outputs.squeeze(0)).reshape(shape)
    batch_y = test_data.inverse_transform(batch_y.squeeze(0)).reshape(shape)

outputs = outputs[:, :, f_dim:]
batch_y = batch_y[:, :, f_dim:]
``` 
该朴素的思路为：将输出张量中的全部数据先进行inverse_transform，随后再切片成想要的输出，之后就不再报错了。
如果我使用不当，还请方便告知正确用法。

**Comment by GuoKaku** [comment]
感谢指正！

**Comment by mwlznb** [comment]
您好，我在使用imputation，feature='MS'的时候，exp_imputation.py也出现mask和output张量不匹配，请问应该如何修改mask的张量，又或者哪个args可以帮助。万分感谢。

**Comment by Gczmy** [comment]
> 您好，我在使用imputation，feature='MS'的时候，exp_imputation.py也出现mask和output张量不匹配，请问应该如何修改mask的张量，又或者哪个args可以帮助。万分感谢。

试试看将exp_imputation.py的第183行
                `masks.append(mask.detach().cpu())`
修改为
               `masks.append(mask[:, :, f_dim:].detach().cpu())`
个人猜测是在MS情况下output和mask选择目标输出时张量形状的不匹配导致的。
